.github/workflows/continuous-integration.yml

```diff
@@ -124,6 +124,7 @@ jobs:
           - "'casper/test:testOnly coop.rchain.casper.engine.*'"
           - "'casper/test:testOnly coop.rchain.casper.genesis.*'"
           - "'casper/test:testOnly coop.rchain.casper.util.*'"
+          - "'node/test:testOnly coop.rchain.node.mergeability.*'"
```

Comment:
move all the mergeability test in the `node`

Reply:
No need to specify test here, it will run with other tests in node project. This also disables running of all other defined tests in node project.
I curious why you moved test to node project? Is it some reference unavailable?

https://github.com/rchain/rchain/blob/a17835b2cb5a6bb25097dd0a74debde0afa6f843/.github/run-unit-test-selection#L4-L11

Reply:
> Is it some reference unavailable?
Yes. `import coop.rchain.node.revvaultexport.RhoTrieTraverser`. Let me remove this line.

Reply:
Should I move `coop.rchain.node.revvaultexport.RhoTrieTraverser` back to casper?